src/spikeinterface/core/base.py

```diff
@@ -926,7 +947,7 @@ def load_extractor(file_or_folder_or_dict, base_folder=None):
         return BaseExtractor.load(file_or_folder_or_dict, base_folder=base_folder)
 
 
-def load_extractor_from_dict(d, base_folder=None):
+def load_extractor_from_dict(d, base_folder=None) -> BaseExtractor:
     print("Use load_extractor(..) instead")
```

Comment:
I think it's better to use `warnings.warn` or `logging.warning` rather than `print`

Reply:
Yep, this is old (not pointint fingers here, but I think it's @samuelgarcia 's fault!)

@h-mayorquin can you make this a `warnings.warn`? 

@DradeAW we don't use logging (but might switch in the future ;)

Reply:
Done.